### PR TITLE
Jonathan development

### PR DESCRIPTION
**Pull request by chanjonathan**
Add text content and image on webpage

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,53 @@
...
+      <h2>Photo Gallery</h2>
+      <img src="images/IMG_6468.JPG" alt="Prince and Heather" width="400" height="300"/>
+      <p>These were my two bunnies Prince and Heather.</p>
+
+      <h2>Projects</h3>
+
+        <h3>Conformer Counter</h3>
+        <ul>
+            <li>A python script that determines whether a methylcyclohexane is axial or equatorial.</li>
+            <li>Will determine the ratio of conformers in the oddly specific scenario that you have thousands of methylcylcohexanes in a .gro file.</li>
```
Comment (EvanJuV) [inline portfolio/src/main/webapp/index.html:28]: Probably a typo in "methylcylcohexanes". Should it be methylcyclohexane?

**Review by EvanJuV — APPROVED** [review]

**Comment by chanjonathan** [comment]
comment

**Review by EvanJuV — CHANGES_REQUESTED** [review]
Very nice Jonathan! 

I think generally speaking, you'd want to merge your PR once it's approved. If you haven't already noticed, the changes you pushed recently were added to this PR. 

Pushing new commits to the same PR makes sense if you're making changes to address comments, but generally you'd like to have one PR per feature. A feature is a relatively subjective term, but in our case we can map it to the work been done each week. In the industry it is usually mapped to an actual feature (e.g. bug fix, feature request, refactor, cleanup, etc.).

Other than that, LGTM! Please push changes addressing comments.

File: portfolio/pom.xml [hunk 2]
```diff
@@ -20,6 +20,13 @@
   </properties>
 
   <dependencies>
+    <!-- Gson -->
+    <dependency>
+    <groupId>com.google.code.gson</groupId>
```
[the remark this answers is not in the document]
Comment (chanjonathan) [reply portfolio/pom.xml:25]: fixed

[the remark this answers is not in the document]
Comment (chanjonathan) [reply portfolio/pom.xml:25]: fixed

**Comment by chanjonathan** [comment]
Fixes applied

**Review by EvanJuV — APPROVED** [review]
LGTM!

**Pull request merged**